src/app/kpis/page.tsx

```diff
@@ -22,7 +22,7 @@ enum Status {
   ERROR = "ERROR",
 }
 
-export default function Kpis() {
+export default function Liquidity() {
```

Comment:
Please keep the naming 
`export default function Kpis() {`

Reply:
i resovled all error